### PR TITLE
Add package: parsing-expect (#56)

### PR DESCRIPTION
**Pull request by markfarrell**
https://github.com/purescript/package-sets/pull/665

### CODE REVIEW

File: new-packages.json [hunk 1]
```diff
@@ -35,4 +35,5 @@
   "purescript-hetu": "https://github.com/je-l/hetu.git",
   "purescript-npm-package-json": "https://github.com/maxdeviant/purescript-npm-package-json.git",
   "purescript-mol-draw": "https://github.com/lukasturcani/mol-draw.git"
+  "purescript-parsing-expect": "https://github.com/markfarrell/purescript-parsing-expect.git"
```
Comment (thomashoneyman) [inline new-packages.json:38]: Missing a comma at the end of the previous line (after mol-draw) so this json is invalid at the moment

**Review by thomashoneyman — CHANGES_REQUESTED** [review]

File: new-packages.json (hunk 1 above)
Comment (markfarrell) [reply new-packages.json:38]: Sorry will fix

Comment (markfarrell) [reply new-packages.json:38]: Added comma at the end of the previous line. Good now?

**Review by thomashoneyman — APPROVED** [review]

**Comment by thomashoneyman** [comment]
I don't have the ability to merge, but this looks good now.

**Pull request merged**